.github/workflows/ci.yml

```
@@ -299,7 +297,51 @@ jobs:
             --env IREE_VULKAN_F16_DISABLE=0 \
             --env IREE_CUDA_DISABLE=0 \
             --env IREE_NVIDIA_GPU_TESTS_DISABLE=0 \
+            --env IREE_NVIDIA_SM80_TESTS_DISABLE=1 \
             --env CTEST_PARALLEL_LEVEL=2 \
+            --env IREE_CTEST_LABEL_REGEX='^requires-gpu|^driver=vulkan$|^driver=cuda$' \
+            --gpus all \
+            --env NVIDIA_DRIVER_CAPABILITIES=all \
+            gcr.io/iree-oss/nvidia@sha256:1717431fd46b8b1e96d95fa72508e3e3eacb5c95f1245b9b7dbeec23ae823d02 \
+            bash -euo pipefail -c \
+              "./build_tools/scripts/check_cuda.sh
+              ./build_tools/scripts/check_vulkan.sh
+              ./build_tools/cmake/ctest_all.sh ${BUILD_DIR}"
+
+  test_sm80:
```

Maybe call this `test_nvidia_sm80`?
```suggestion
  test_nvidia_sm80:
```

Also remember to add to the `summary` job.